### PR TITLE
Adapt to new registration, add filter for participant tags

### LINKED ISSUE #1414: Make sure the registration process works as we like
- [x] template json should be adapted on website /registration
- [x] explain that this is JSONC
- [x] adapt template.json, to show the new field names 
- [x] remove `I agree to wear my mask to protect everyone as much as possible`
- [x] add `I asked my company about sponsoring` and link the sponsoring board
- [x] remove `I will come and have tested and be covid-negative, at least 24h before I join the event`

### PR DESCRIPTION
**Pull request by Narigo**
Resolves #1414

### CODE REVIEW

**Comment by lowsky** [comment]
I have the feeling we have the information twice, or redundant.

I suggest to make it easier to read: How about removing that info from the json-template, while the info is also below in the field descriptions below:

If not, please add two comments on the `twitter` and `mastodon`  field.
Btw. Isn't there the `website` missing?
... I will add a comment to the code...

![image](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiNDI5ZTM3ZWEtNWE4NS00NWQ1LTg4NDUtZTNkYWQ2MWU2M2JlIiwiZGVwbG95X2lkIjoiNjQ0ZTcyOWNiMTkyMTIwMDA4MTQ2MTE0IiwiaWQiOiJlZDYyMGM5MC0yMjExLTQ0ZjctYWUzZC0yYTUwNGI1Y2E5ZDQifQ==
<details><summary>Browser metadata</summary>





```
Path:      /registration
Browser:   Firefox 113.0 on Mac OS 10.15
Viewport:  1824 x 1017 @1.5789473684210527x
Language:  de-DE
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1422--jscraftcamp.netlify.app%2Fregistration&os=OS%20X&os_version=Catalina&browser=Firefox&browser_version=113.0&resolution=1824x1017&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-1422--jscraftcamp.netlify.app/registration) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=644e729cb192120008146114&commentID=644e92e42e14226a940e5e5c&resolution=resolved)

File: src/routes/registration/+page.svelte [hunk 1]
```diff
@@ -25,20 +26,37 @@
...
+    "whatCanIContribute": "???",
+    // if you want a T-Shirt we need your size and variant preference (optional)
+    "tShirt": {
+        // S | M | L | XL | XXL
+        "size": "S",
+        // fitted (also known as waist cut or women variant) or regular
+        "type": "fitted"
+    },
+    "twitter": "yourhandle",
+    "mastodon": "your Mastodon URL"
```
Comment (lowsky) [inline src/routes/registration/+page.svelte:59]: Could we add the comments to make it clear that they are optional, like others, above?

How about:
```suggestion
    // optional
    "website": "URL - your home page or website",
    // optional
    "twitter": "yourhandle",
    // optional 
    "mastodon": "your Mastodon URL"
```

**Review by lowsky — COMMENTED** [review]
Overall looks, pretty good, could you have a look into the suggested minor adaption on the template on the registration page?

**Review by lowsky — APPROVED** [review]

**Comment by Narigo** [comment]
Thanks for the feedback and yes, I forgot to add the website field here! I've added `required` and `optional` as comments to each field now instead of the longer comment. It should be explained below already.

Are you good with that @lowsky ?

**Comment by lowsky** [comment]
Looks pretty good 👍

**Pull request merged**